handlers/announcement.go

```diff
@@ -118,73 +123,129 @@ func parseSpan(a *html.Node) (text string, link string) {
 	return text, link
 }
 
-// GetAnnouncements returns all the announcements from a specified URL.
+// scrapeAnnouncements returns all the announcements from a specified URL.
 //
-// It is a handler function which scrapes the URL and retrieves elements
+// It scrapes the URL and retrieves elements
 // to convert them into the Announcement type.
-func GetAnnouncements() http.HandlerFunc {
-	return func(w http.ResponseWriter, r *http.Request) {
-		// Request the HTML page
-		response, err := http.Get("http://nitkkr.ac.in/sub_courses.php?id=80&id4=52")
-		if err != nil {
-			RespondError(w, 404, "The source web-page for scraping was not found")
-			return
-		}
-		defer response.Body.Close()
-		if response.StatusCode != 200 {
-			RespondError(w, response.StatusCode, "")
-			return
-		}
+func scrapeAnnouncements() (announcements []Announcement) {
+	// Request the HTML page
+    response, err := http.Get("https://nitkkr.ac.in/?page_id=621")
+	if err != nil {
+		//RespondError(w, 404, "The source web-page for scraping was not found")
+		return announcements
+	}
+	defer response.Body.Close()
+	if response.StatusCode != 200 {
+		//RespondError(w, response.StatusCode, "")
+		return announcements
+	}
 
-		// Load the HTML document
-		doc, err := goquery.NewDocumentFromReader(response.Body)
-		if err != nil {
-			RespondError(w, 502, "Server could not parse the source HTML document")
-			return
-		}
+	// Load the HTML document
+	doc, err := goquery.NewDocumentFromReader(response.Body)
+	if err != nil {
+		//RespondError(w, 502, "Server could not parse the source HTML document")
+		return announcements
```

Comment:
I don't fully recall Go's return variable thingy, but thinking you can just here.

```suggestion
		return
```